sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -249,26 +249,26 @@ public void testStatementBuilder()
     public void testStatementBuilderTpch()
             throws Exception
     {
-        printTpchQuery(1, 3);
+        // printTpchQuery(1, 3); interval is not supported
         printTpchQuery(2, 33, "part type like", "region name");
         printTpchQuery(3, "market segment", "2013-03-05");
-        printTpchQuery(4, "2013-03-05");
-        printTpchQuery(5, "region name", "2013-03-05");
-        printTpchQuery(6, "2013-03-05", 33, 44);
+        // printTpchQuery(4, "2013-03-05"); interval is not supported
+        // printTpchQuery(5, "region name", "2013-03-05"); interval is not supported
```

Comment:
can the queries be adjusted to make sense without interval support instead of removing them completely?